From f99e24a6778a065dad732b916b2648352609c79a Mon Sep 17 00:00:00 2001
From: Thomas Ebeling <penguins@bollie.de>
Date: Fri, 29 May 2020 19:32:56 +0200
Subject: [PATCH] ALSA: usb-audio: Fixing usage of plain int instead of NULL
Git-commit: f99e24a6778a065dad732b916b2648352609c79a
Patch-mainline: v5.8-rc1
References: git-fixes

As reported by kbuild test robot, mixer quirks for RME Babyface
Pro used plain integer instead of NULL.

Fixes: 3e8f3bd04716 ("ALSA: usb-audio: RME Babyface Pro mixer patch")
Signed-off-by: Thomas Ebeling <penguins@bollie.de>
Reported-by: kbuild test robot <lkp@intel.com>
Link: https://lore.kernel.org/r/20200529173248.zzawijfvw73kzjxt@bollie.ca9.eu
Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 sound/usb/mixer_quirks.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/usb/mixer_quirks.c b/sound/usb/mixer_quirks.c
index aad2683ff793..b6bcf2f92383 100644
--- a/sound/usb/mixer_quirks.c
+++ b/sound/usb/mixer_quirks.c
@@ -2255,7 +2255,7 @@ static int snd_bbfpro_ctl_update(struct usb_mixer_interface *mixer, u8 reg,
 	err = snd_usb_ctl_msg(chip->dev,
 			      usb_sndctrlpipe(chip->dev, 0), usb_req,
 			      USB_DIR_OUT | USB_TYPE_VENDOR | USB_RECIP_DEVICE,
-			      usb_val, usb_idx, 0, 0);
+			      usb_val, usb_idx, NULL, 0);
 
 	snd_usb_unlock_shutdown(chip);
 	return err;
@@ -2394,7 +2394,7 @@ static int snd_bbfpro_vol_update(struct usb_mixer_interface *mixer, u16 index,
 			      SND_BBFPRO_USBREQ_MIXER,
 			      USB_DIR_OUT | USB_TYPE_VENDOR |
 			      USB_RECIP_DEVICE,
-			      usb_val, usb_idx, 0, 0);
+			      usb_val, usb_idx, NULL, 0);
 
 	snd_usb_unlock_shutdown(chip);
 	return err;
-- 
2.16.4

